packages/lab/src/DotPagination/DotPagination.js

```diff
@@ -38,22 +38,25 @@ const HvDotPagination = (props) => {
   const icons = getSelectorIcons(classes, {});
 
   return (
-    <div className={clsx(className, classes.root)} {...others}>
-      <HvRadioGroup classes={{ horizontal: classes.horizontal }} orientation="horizontal">
-        {range(pages).map((pg, i) => (
-          <HvRadio
-            classes={{ root: classes.radioRoot }}
-            key={`dot_navigation${id && id.toString()}_item${i.toString()}`}
-            value={i}
-            checked={page === i}
-            onChange={() => onPageChange(i)}
-            icon={icons.radio}
-            checkedIcon={icons.radioChecked}
-            aria-label={getItemAriaLabel(i)}
-          />
-        ))}
-      </HvRadioGroup>
-    </div>
+    <HvRadioGroup
+      className={clsx(className, classes.root)}
+      classes={{ horizontal: classes.horizontal }}
+      orientation="horizontal"
+      {...others}
+    >
+      {range(pages).map((i) => (
+        <HvRadio
+          classes={{ root: classes.radioRoot }}
+          key={i}
+          value={i}
+          checked={page === i}
+          onChange={() => onPageChange(i)}
```

Comment:
This onPageChange could also bubble up the Change event to the user 

`onChange={() => onPageChange(i)}` -> `onChange={(event) => onPageChange(event, i)}`